ONSdigital/django-oauth2-test
# Performance improvement 650 concurrency

# Motivation and Context
Performance Improvement

# What has changed
- Changing Gunicorn `sync` workers from 4 to 8.
- Requires 512M instance on cloud foundry environments.
- Adding persistent DB connections for cloud foundry environments.
- Upgrading `cryptography` package to `1.9` to remove `setuptools` dependency. This problem manifest only in the performance test environment where the python build package requires all dependencies to be pinned to exact version. 

# How to test?
- Has been load tested in Prunes. Check the deployment pipeline is green after the changes are merged to `master`.
- On Pre-Prod (which uses the same python build package as Prod), execute the `create-account`, `verify-account` and `sign-in` actions manually and check the system works as expected.

# Links
NA

# Screenshots (if appropriate):
